shell/packages/sandstorm-db/migrations.js

```diff
@@ -784,6 +821,9 @@ function migrateToLatest() {
 
       Migrations.update({ _id: "new_server_migrations_applied" }, { $set: { value: true } });
     }
+
+    // Start background migrations.
+    backgroundFillInGrainSizes();
```

Comment:
Ditto - how about this goes in the `Meteor.startup()` block in `server/00-startup.js`?